WordPressShared/WordPressShared/Core/Utility/NSString+Helpers.m

```diff
@@ -112,6 +112,14 @@ + (NSString *)stripShortcodesFromString:(NSString *)string
                                       withTemplate:@""];
 }
 
++ (NSString *)summaryFromContent:(NSString *)string
```

Comment:
I agree with factoring `summaryFromContent` out of `BasePost` but I'm not sure its generic enough to be a good addition to an `NSString` category/extension.  Its usecase is pretty specific.  The logic might have a better home in `ReaderPostServiceRemote createSummaryFromContent` and then we could just do away with the method.  As far as I can tell its not being called from anywhere else....